packages/web3-validator/src/validation/bytes.ts

```diff
@@ -16,26 +16,21 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { ValidInputTypes } from '../types';
-import { parseBaseType } from '../utils';
+import { hexToUint8Array, parseBaseType } from '../utils';
 import { isHexStrict } from './string';
 
 /**
  * checks input if typeof data is valid buffer input
  */
-export const isBuffer = (data: ValidInputTypes) => Buffer.isBuffer(data);
+export const isBuffer = (data: ValidInputTypes) => data instanceof Uint8Array;
```

Comment:
`isBuffer` should be removed as we should be accepting Uint8Array inputs only,
 like here https://github.com/web3/web3.js/blob/2258b2cbb4a7c9e15cbe6dbd07d29f10e4da9350/packages/web3-eth-accounts/src/account.ts#L82